webpack/components/extensions/HostDetails/Tabs/__tests__/packagesTab.test.js

```diff
@@ -141,3 +165,363 @@ test('Can filter by package status', async (done) => {
   assertNockRequest(autoSearchScope);
   assertNockRequest(scope2, done); // Pass jest callback to confirm test is done
 });
+
+test('Can select one package', async (done) => {
```

Comment:
I do hope it wasn't too much work adapting all these tests from ErrataTab.  All of the tests you've added here aren't really specific to Packages. Rather, they test the Select All functionality, and so I don't think we need to duplicate them everywhere. I'll leave it up to you whether to delete them or not, but if they weren't there I would not ask for them to be added.  Perhaps it would have helped if the existing tests were in a "describe" block:
```js
describe('Select All functionality'), () =>
  test('Can select one', async (done) => {
...
```